frontend/src/app/resource-form/form-image/form-image.component.html

```diff
@@ -4,8 +4,7 @@ <h3>
     Image
   </h3>
   <p>
-    A starter Jupyter Docker Image with a baseline deployment and typical ML
-    packages.
+    Select the base software you want installed in your server. If you choose a custom image, it must be available in the DAaaS container registry.
```

Comment:
Check formatting with Prettier, which should wrap this line.

Reply:
Changes done